website/docs/tutorials/08-interacting-with-starknet/01-invoke.md

```diff
@@ -0,0 +1,37 @@
+# Invoking contracts
+
+## Overview
+This cheatcode allows you to send an invoke transaction with `@external` function entrypoint.
```

Comment:
Cheatcode?

Reply:
[Correct wording. Change int to hex](https://github.com/software-mansion/protostar/pull/1063/commits/990e8d986c4bd18f1378261a2b57db17f1797408)